user-info-provider/src/main/java/de/intension/mapper/oidc/UserInfoProviderMapper.java

```diff
@@ -123,13 +153,43 @@ private static void addOnlyChildrenClaims()
         configProperties.add(property);
     }
 
+    private static void addAttributesBasedOnRole()
+    {
+        ProviderConfigProperty property = new ProviderConfigProperty();
+        property.setName(PROFESSIONAL_ROLES);
+        property.setLabel("Professional roles");
```

Comment:
why the "professional"? :shrug:

Reply:
To give more details about the roles. It is also mentioned in the ticket.